packages/icon/src/vaadin-icon.js

```diff
@@ -110,7 +110,12 @@ class Icon extends ThemableMixin(ElementMixin(ControllerMixin(SlotStylesMixin(Ic
         viewBox="[[__computeViewBox(size, __viewBox)]]"
         preserveAspectRatio="[[__computePAR(__defaultPAR, __preserveAspectRatio)]]"
         aria-hidden="true"
-      ></svg>
+      >
+        <g id="svg-group"></g>
+        <g id="use-group" hidden="[[!__useRef]]">
```

Comment:
Looks like the `hidden` here doesn't do anything. Should be hidden somehow else (if necessary)

Reply:
Don't forget to push